core-bundle/assets/modules/turbo-stream-connection.js

```diff
@@ -0,0 +1,85 @@
+export class TurboStreamConnection {
+    _abortController = null;
+    _abortSignal = { reason: 'The request was substituted.' }
+
+    /**
+     * Requests a stream response using GET and lets Turbo handle it.
+     *
+     * @param url The URL of the Symfony controller answering the stream request.
+     * @param query_params An object of query parameters. If the value is an array, a key "foo" will be named "foo[]" and appear multiple times.
+     * @param abortPending If set to true, previous requests that are still pending will be aborted.
+     *
+     * @returns {Promise<void>}
+     */
+    async get(url, query_params = null, abortPending = false) {
+        let params = {
+            method: 'get',
+            headers: {
+                'Accept': 'text/vnd.turbo-stream.html',
+            }
+        };
+
+        if (abortPending) {
+            if (null !== this._abortController) {
+                this._abortController.abort(this._abortSignal);
+            }
+
+            this._abortController = new AbortController();
+
+            params = {
+                ...params,
+                signal: this._abortController.signal,
+            }
+        }
+
+        let response;
+
+        try {
+            response = await fetch(TurboStreamConnection.buildURL(url, query_params), params);
```

Comment:
```suggestion
            response = await fetch(this.constructor.buildURL(url, query_params), params);
```
Otherwise if I extend from `TurboStreamConnection` overwriting `buildURL()` would not have an effect.
Sadly JavaScript does not seem to have a keyword for this use case (like PHPs `static::`).

This is of course just a nitpick and can be merged either way ☺️

Reply:
Learned something new today. :slightly_smiling_face: 
Done in 6791e540c180f9c1c3c59504725b2816052df0bf.